### ISSUE TITLE
SCP1905 - AscAdiposeProgenitor

### ISSUE THREAD

**Issue by arschat** [ebi-ait/hca-ebi-wrangler-central#1342]
### Project short name:
AscAdiposeProgenitor

### Primary Wrangler:
Arsenios

### Associated files

* Google Drive:

### Published study links

* Paper: [Single-nuclei Transcriptome of Human AT Reveals Metabolically Distinct Depot-Specific Adipose Progenitor Subpopulations](https://www.biorxiv.org/content/10.1101/2022.06.29.496888v1) - 10.1101/2022.06.29.496888

* Accessioned data: [SCP1905](https://singlecell.broadinstitute.org/single_cell/study/SCP1905/human-preadipocyte-single-cell-analysis), [SCP1903](https://singlecell.broadinstitute.org/single_cell/study/SCP1903/human-adipose-tissue-single-nuclear-rna-sequencing)
* ingest: [3294cc9d-07ce-4a18-8e5f-a8afac5c882e](https://contribute.data.humancellatlas.org/projects/detail?uuid=3294cc9d-07ce-4a18-8e5f-a8afac5c882e&tab=project)

### Key Events

- [ ] Convert published metadata to HCA spreadsheet
- [ ] Manually curate dataset to meet HCA metadata standard
- [ ] Collect any matrix and cell-type annotation files
- [ ] Are the analysis files suitable for CellxGene? If something is missing get in touch with the authors to request it
- [ ] Upload sheet to validate metadata
- [ ] Transfer raw files to ingest to validate data files
- [ ] Check linking using ingest graph validator
- [ ] Ask the Secondary Wrangler for an end-to-end review of the project. Ask the Expertise Wrangler to review specific tabs if needed
- [ ] Submit dataset to Production 
- [ ] Complete the Export SOP
- [ ] Convert project data to SCEA format following the SCEA conversion SOP if appropriate

**Comment by arschat** [comment]
In publication it is mentioned that tissue was frozen
> Nuclei isolation from human VAT and SAT was adapted from a previous protocol.[19](https://www.biorxiv.org/content/10.1101/2022.06.29.496888v1.full#ref-19) Briefly, 400-500mg samples of **frozen** human VAT or SAT were washed with sterile RNAse-free cold 1X DPBS and minced with a scalpel in a petri dish on ice.

Storage method is specified in [referenced protocol paper](https://www.nature.com/articles/nprot.2016.015)
> Nuclear transcriptomes can be obtained from postmortem human brain tissue stored at −80 °C, making brain archives accessible for RNA-seq from individual neurons.

**Comment by arschat** [comment]
hca-util upload area `0adb81a9-6fa9-486c-8318-274427c55bd6`

**Comment by arschat** [comment]
If we make some assumptions to avoid blockers, we have a graph valid submission.
Assumptions:
- lib_prep is `10x 3' v3.1`
- genome version is `GRCh38`
- medications is 
    - insulin: [insulin isophane / insulin, regular, human Pen Injector](https://www.ebi.ac.uk/ols4/ontologies/dron/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FDRON_00730920)
    - metformin: [metformin Oral Solution](https://www.ebi.ac.uk/ols4/ontologies/dron/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FDRON_00027969)
    - Statin: [statin (semi-synthetic)](https://www.ebi.ac.uk/ols4/ontologies/dron/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FCHEBI_87633)

**Comment by idazucchi** [comment]
I agree with the assumptions you've made, but also agree with not releasing this on the portal unless we have no other way of moving forward

### Donor
- I agree on using the sex from the publication for donor 2. But it's very weird - donor 4 is also affected, and half of the samples are m and half are f
- there are discrepancies in donor ages and BMI 
- donor 4 has hypertension, donor 3 doesn't have hypertension, donor 6 isn't pre-diabetic (but does take insulin? maybe there is a mistake in the table?) but has sleep apnea, donor 7 is only pre-diabetic, 
- donor 8 doesn't take insulin, donor 9 doesn't take statin
- Can HbA1c levels go in the test result section? 
- ethnicity is unknown for donor 9
- for [metformin](https://www.ebi.ac.uk/ols4/ontologies/dron/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FCHEBI_6801?lang=en) and [statin](https://www.ebi.ac.uk/ols4/ontologies/dron/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FCHEBI_87631) - there are the CHEBI terms if you want

### Specimen
- for organ part I see you tried to map the exact sampled location, maybe we can add this as well? [VAT](https://www.ebi.ac.uk/ols4/ontologies/uberon/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FUBERON_0035818), [SAT](https://www.ebi.ac.uk/ols4/ontologies/uberon/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FUBERON_0002190)
- review disease based on comments aboove
### Cell suspension
- trypan blue only for cultured cells :0

**Comment by arschat** [comment]
Thanks a lot for the detailed review. Donor 6 not being a diabetic is probably a typo, since the way the decided about the diabetes is the level of HbA1c:
> Subjects with type 2 diabetes (DM) were defined by clinical diagnosis requiring medication and hemoglobin A1c (HbA1c)>=6.5%. Non-diabetic (NDM) subjects were defined by no clinical history of diabetes and HbA1c<5.7% per American Diabetes Association criteria.

Donor 6's HbA1c is 5.7 while donor 9 who is considered diabetic, is 5.8.

**Comment by arschat** [comment]
Dataset to be put on hold for publication (see blockers in comment above) but ready to send as Tier 1 to contributor.
If contributor does not reply within two months we will reconsider publishing.
